upgrading/automated_upgrades.adoc

```diff
@@ -782,25 +782,22 @@ other nodes in their class (`app` versus `infra`), one at a time.
 .. Run `oc get daemonset` to verify the label found under `NODE-SELECTOR`. The
 default value is `storagenode=glusterfs`.
 
-.. Remove the daemonset label from the node:
+.. Add a label (for example, `type=upgrade`) to the node you want to upgrade.
 +
+[NOTE]
```

Comment:
Just wondering if this has to be in a note? If everyone coming through these steps would use the upgrade_nodes.yaml playbook, I'd assume it'd just be part of this step. Or if it is necessary as a note, maybe reword the intro sentence of the note ("Use the upgrade_nodes.yaml playbook") to be clearer why it's a note. Is it only in certain situations? Is this just an example?

Reply:
Good point. I took it out of a note. Step is now using that playbook to run the listed command. The note identifying the location of the playbook now follows that step.